clients/ui/frontend/src/app/pages/modelCatalog/components/ModelCatalogCard.tsx

```diff
@@ -48,9 +53,18 @@ const ModelCatalogCard: React.FC<ModelCatalogCardProps> = ({ model, source, trun
             )}
             <FlexItem align={{ default: 'alignRight' }}>
               {isValidated ? (
-                <Label color="purple">Validated</Label>
+                <Popover bodyContent={MODEL_CATALOG_POPOVER_MESSAGES.VALIDATED}>
+                  <Label color="purple">Validated</Label>
+                </Popover>
               ) : (
-                source && <Label>{source.name}</Label>
+                source &&
+                (source.name === ModelCatalogProvider.RED_HAT ? (
```

Comment:
Red Hat related things should be done midstream since it's a small test change - it needn't be an extension